extensions/core/interfaces/calendar/readonly.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+  <div class="no-wrap">{{ value }}</div>
```

Comment:
I think it would be nice if there's an option to set the value to "relative" mode, just like in the `date` interface 😄